wnfs/src/common/blockstore.rs

```diff
@@ -28,40 +27,17 @@ pub trait BlockStore {
         self.put_block(bytes, IpldCodec::DagCbor).await
     }
 
-    async fn put_private_serializable<V: Serialize>(
```

Comment:
I removed these. Since I added `SnapshotKey::encrypt` and `TemporalKey::key_wrap_encrypt`, we weren't making use of a single `put_private_serializable` anymore.
We *could* add two more functions for en/decrypting on the `BlockStore` trait, but it felt like it'd balloon in size at some point and have quite some dependencies.